services/director-v2/src/simcore_service_director_v2/modules/docker_registry.py

```diff
@@ -58,7 +58,10 @@ def __init__(self, settings: RegistrySettings):
         # TODO: add auth https://www.python-httpx.org/advanced/#customizing-authentication
         # TODO: see https://colin-b.github.io/httpx_auth/
 
-        self.client = AsyncClient(base_url=self.settings.api_url)
+        self.client = AsyncClient(
```

Comment:
Same as other comment. use settings instance in ``app.state.settings``

Reply:
changed